docs/source/hsm.md

```diff
@@ -89,31 +108,31 @@ in the `signcerts` folder to retrieve the private key from inside the HSM.
 The process for creating the MSP folders will be different depending on if you
 are using a Fabric Certificate Authority (CA) your own CA.
 
-### Using a Fabric CA
+### Using an HSM with a Fabric CA
+
+You can set up a Fabric CA to use an HSM by making the same edits to the CA server configuration file as you would make to a peer or ordering node. Because you can use the Fabric CA to generate keys inside an HSM, the process of creating the local MSP folders is straightforward. Use the following steps:
+
+1. Create an HSM token and point to it in the Fabric CA server configuration file. When the Fabric CA server starts, the HSM described in the configuration file will generate the CA signing certificate and store the private key inside the HSM. If you are not concerned about exposing your CA signing certificate, you can skip this step and only configure an HSM for your peer or ordering nodes.
```

Comment:
Replace HSM token with Label. Also the precondition step should be : 

1. Create Partition on HSM Server and note down Label and PIN for partition.

1.  Follow HSM Vendor document to configure HSM Client and Server to communicate.

Reply:
Added a Before you begin section with these steps.